creator-node/src/services/stateMachineManager/stateReconciliation/issueSyncRequest.jobProcessor.js

```diff
@@ -110,13 +113,25 @@ module.exports = async function ({ logger, syncType, syncRequestParameters }) {
     logger.error(`${logMsgString} || Error issuing sync request: ${e.message}`)
   }
 
+  const metricsToRecord = []
+
   // Wait until has sync has completed (within time threshold)
-  const additionalSyncIsRequired = await _additionalSyncIsRequired(
-    userWallet,
-    primaryClockValue,
-    secondaryEndpoint,
-    syncType,
-    logger
+  const startWaitingForCompletion = Date.now()
+  const { additionalSyncIsRequired, reasonForAdditionalSync } =
+    await _additionalSyncIsRequired(
+      userWallet,
+      primaryClockValue,
+      secondaryEndpoint,
+      syncType,
+      logger
+    )
+  metricsToRecord.push(
+    makeHistogramToRecord(
+      'ISSUE_SYNC_SECONDS_WAITING_FOR_COMPLETION_HISTOGRAM',
```

Comment:
why not pass by reference eg https://github.com/AudiusProject/audius-protocol/blob/theo-add-sync-metric/creator-node/src/routes/tracks.js#L309

Reply:
I changed the other part you mentioned above to reference the actual var in MetricNames, but in this case I don't see what else I could pass by reference since this is in the job processor and can only return plain objects